src/controller/python/chip/clusters/Attribute.py

```diff
@@ -323,6 +472,15 @@ def _OnReadAttributeDataCallback(closure, endpoint: int, cluster: int, attribute
         EndpointId=endpoint, ClusterId=cluster, AttributeId=attribute), status, dataBytes[:])
 
 
+@_OnReadEventDataCallbackFunct
+def _OnReadEventDataCallback(closure, endpoint: int, cluster: int, event: int, number: int, priority: int, timestamp: int, timestampType: int, data, len):
+    dataBytes = ctypes.string_at(data, len)
+    path = EventPath(ClusterId=cluster, EventId=event)
+    eventType = _EventIndex.get(str(path), None)
```

Comment:
This _can_ fail. We should do something similar to what we do for attributes:

```
            attributeType = _AttributeIndex.get(str(AttributePath(
                ClusterId=path.ClusterId, AttributeId=path.AttributeId)), None)
            attributeValue = None
            tlvData = chip.tlv.TLVReader(data).get().get("Any", {})
            if attributeType is None:
                attributeValue = ValueDecodeFailure(
                    tlvData, LookupError("attribute schema not found"))
```

That should happen in `handleEventData`. Consequently, you should:
1. Avoid initializing `Event` in the `EventHeader` built below.
2. Pass in the event ID into `handleEventData` to permit it to properly do the look up.

Reply:
thanks, just update, pass EventPath into handleEventData to permit it to properly do the look u